knip.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://unpkg.com/knip@5/schema.json",
+  "ignore": [
+    "apps/**/generated/**",
+    "packages/**/generated/**"
```

Comment:
question: should we add node__modules to ignored?

Reply:
no, knip has defaults that points to entry points so node models are not scanned as a root